packages/core/nuxt-theme-module/README.md

```diff
@@ -0,0 +1,43 @@
+# Nuxt Theme Module
+This module:
+- Changes `.nuxt` buildDir to name you set in the config or `.theme` 
+- Adds `reset.scss` transpiled to CSS to the head (it removes default stylings applied by browser, e.g. blue links)
+- By default, it adds eCommerce routes to your app.
+- If you set `generate` in the config, it will copy, merge & watch for changes in Agnostic and Integration theme. 
+
+## How to install?
+Install module:
+```sh
+yarn add @vue-storefront/nuxt-theme
+```
+Add it to build modules in your `nuxt.config.js`:
+```js
+['@vue-storefront/nuxt-theme'],
+```
+
+## Configuration details
+If you want to disable autoadding routes you can do it:
+```js
+['@vue-storefront/nuxt-theme', {
+    routes: false
+}],
+```
+To properly configure `generate` property you have to provide data that will be replaced in EJS templates:
```

Comment:
Would be nice to mention that this property applies only to core development as well as an example of such EJS template (just an import)